apps/client/src/pages/index.tsx

```diff
@@ -78,20 +89,24 @@ export default function IndexPage() {
               </div>
             </div>
             <div className="flex  max-w-lg  gap-8">
-              <div>Icon</div>
               <div>
-                <h3 className="mb-4">Feedback and critique</h3>
+                <Image src={learning} alt="feeback" className='w-[40px]'/>
+              </div>
+              <div>
+                <h3 className="mb-4 text-slate-100 text-xl font-bold capitalize">Feedback and critique</h3>
                 <p>
                   Get constructive feedback and critiques from other writers and
                   experts in your niche to help you improve your skills and grow
                   as a blogger.
                 </p>
               </div>
             </div>
-            <div className="flex  max-w-lg  gap-8">
-              <div>Icon</div>
+            <div className="flex max-w-lg  gap-8">
+              <div>
+                <Image src={community} alt="feeback" className='w-[40px]'/>
+              </div>
               <div>
-                <h3 className="mb-4">Feedback and critique</h3>
+                <h3 className="mb-4 text-slate-100 text-xl font-bold capitalize">Feedback and critique</h3>
                 <p>
                   Get constructive feedback and critiques from other writers and
```

Comment:
```suggestion
                  Participate in forums and discussion groups to connect with other writers and bloggers, share your work, and get feedback.
```

Reply:
Okay noted 

The keyboard of my PC is currently damaged I'll resolve it when i fix it.